guide/glossary/node.md

```diff
@@ -0,0 +1,219 @@
+   image = "/assets/images/guide/glossary/node/local-node.png"
+   retina = "/assets/images/guide/glossary/node/local-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+A very simple setup is to use an application like Bitcoin Core that combindes node and wallet functionality.
+
+</div>
+
+#### Hardware node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/hardware-node.png"
+   retina = "/assets/images/guide/glossary/node/hardware-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Separating node and wallet functionality, many users rely on a dedicated computer like a Rasperry Pi running software like Umbrel, MyNode, and Raspiblitz. Wallet applications can connect to the node which allows them to be more lightweight.
+
+</div>
+
+#### Self-hosted node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/own-hosted-node.png"
+   retina = "/assets/images/guide/glossary/node/own-hosted-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+While it possible to run a node on a rented server, this partially places the node in the control of a third party and introduces privacy risks.
+
+</div>
+
+#### Third-party node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/third-party-node.png"
+   retina = "/assets/images/guide/glossary/node/third-party-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+A convenient setup is to connect to a node controlled by a third-party, which is only recommended if there is a good trust relationship.
+
+</div>
+
+#### Multiple third-party nodes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/multiple-third-party-nodes.png"
+   retina = "/assets/images/guide/glossary/node/multiple-third-party-nodes@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Connecting to multiple untrusted third-parties can be a good idea if compact block filters are used. This prevents third parties from analyzing which data belongs to the user, an important privacy benefit. It also reduces the risk of receiving manipulated data from a single third-party node, as data from multiple sources can be compared.
+
+</div>
+
+#### Third-party API
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/third-party-api.png"
+   retina = "/assets/images/guide/glossary/node/third-party-api@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Third-party APIs should be avoided. These are custom connection mechanisms built on top of block data, and there is no way to validate the authenticity of the data.
```

Comment:
This one can also be seen to contradict the previous heading. At least a reorder could help.

> Connecting to multiple untrusted third-parties can be a good idea if compact block filters are used

> Third-party APIs should be avoided. These are custom connection mechanisms built on top of block data, and there is no way to validate the authenticity of the data.

Reply:
The difference is API vs. node. A node sends you actual block data while an API can modify and reorganize data and send you something different. Does that address your concern? Or did I not understand correctly?

Reply:
Also these two headings specifically touch on types of node connections vs setup.

Reply:
I flipped the order slightly so the options we don't recommend are at the bottom. Let me know if that addresses your feedback.